MapboxNavigationTests/NSExpressionTests.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+@testable import MapboxNavigation
+
+class NSExpressionTests: XCTestCase {
+    func testLocalization() {
+        let jsonExpression: [Any] = [
+            "step",
+            ["zoom"],
+            [
+                "case",
+                [
+                    "<",
+                    [
+                        "number",
+                        ["get", "area"]
+                    ],
+                    80000
+                ],
+                ["get", "abbr"],
+                ["get", "name_en"]
+            ],
+            5, ["get", "name_en"]
+        ]
+        let localizedJSONExpression: [Any] = [
+            "step",
+            ["zoom"],
+            [
+                "case",
+                [
+                    "<",
+                    [
+                        "number",
+                        ["get", "area"]
+                    ],
+                    80000
+                ],
+                ["get", "abbr"],
+                ["get", "name"]
+            ],
+            5, ["get", "name"]
+        ]
+        let expression = NSExpression.mgl_expression(withJSONObject: jsonExpression)
+        let localizedExpression = expression.localized(into: nil)
+        XCTAssertEqual(localizedExpression.mgl_jsonExpressionObject as! NSArray, localizedJSONExpression as NSArray)
```

Comment:
This test will fail until mapbox/mapbox-gl-native#11615 lands in a build of the map SDK.